workflows/metatdenovo.nf

```diff
@@ -252,7 +250,7 @@ workflow METATDENOVO {
         PROKKA_CAT(ch_assembly_contigs)
         ch_versions = ch_versions.mix(PROKKA_CAT.out.versions)
         ch_gff      = PROKKA_CAT.out.gff.map { it[1] }
-        ch_hmm_aa   = PROKKA_CAT.out.faa
+        //ch_hmm_aa   = PROKKA_CAT.out.faa
```

Comment:
Do we need to remove this comment? there are other comments inside this script. was it a test?

Reply:
I forgot to mention this when we spoke. I couldn't see  a reason why there are two channels for amino acid output, but I didn't dare to just remove so I commented them out. I'll run some tests with the other orf callers to see that all works, then remove the commented lines.